sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -49,6 +49,24 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession with AdaptiveSpark
 
   setupTestData()
 
+  test("lpad/rpad should throw an exception for invalid length input if the ANSI mode enabled") {
```

Comment:
Thank you, @maropu . 
Can we add these new test coverage into `string-functions.sql` instead of `SQLQuerySuite`?

Reply:
For `ansi`, you can create a new `string-functions.sql` under `ansi` directory.

Reply:
ok